tools/dependency_summary.py

```diff
@@ -982,7 +987,7 @@ def pick_most_acceptable_license(self, id, licenseId):
         """
         # Split "A/B" and "A OR B" into individual license names.
         licenses = set(l.strip()
-                       for l in re.split(r"\s*(?:/|\sOR\s)\s*", licenseId))
+                       for l in re.split(r"\s*(?:/|\sOR\s)\s*", str(licenseId)))
```

Comment:
for some reason one (or more?) of the licenseIds was not a string after the `cargo update` 🤷 I added this and all seems to work properly

Reply:
nvm me, it was the same license that needed the fixup 🤦